autoload/codefmt.vim

```diff
@@ -454,3 +516,17 @@ endfunction
 function! codefmt#SetWhetherToPerformIsAvailableChecksForTesting(enable) abort
   let s:check_formatters_available = a:enable
 endfunction
+
+""
+" @private
+" Returns all formatters, for debugging.
```

Comment:
Tab completion on `:FormatCode` gets you the list of all formatters (with applicable formatters first). If we need more info for debugging, could we add a `:CodeFormatInfo` command to help with that and user troubleshooting?